kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java

```diff
@@ -458,24 +468,14 @@ public long getLongProperty(String name) throws EntityPropertyNotDefinedExceptio
 		}
 	}
 
-	/**
-	 * Access a named property as a Time.
-	 * 
-	 * @param name
-	 *        The property name.
-	 * @return the property value
-	 * @exception EmptyException
-	 *            if not found.
-	 * @exception TypeException
-	 *            if the property is found but not a Time.
-	 */
+	@Override
 	public Time getTimeProperty(String name) throws EntityPropertyNotDefinedException, EntityPropertyTypeException
 	{
 		String p = getProperty(name);
 		if (p == null) throw new EntityPropertyNotDefinedException();
 		try
-		{
-			return TimeService.newTimeGmt(p);
+		{	TimeService timeService = ComponentManager.get(TimeService.class);
```

Comment:
Format's a bit odd here. Also, you may as well just put those on one line.

Reply:
OK